src/components/about/About.module.scss

```diff
@@ -0,0 +1,18 @@
+@use '../../styles/variables';
+
+.aboutContainer {
```

Comment:
Follow the prettier styling. Can set your IDE to follow the prettierrc rules and automatically format the code for you.